src/julia.h

```diff
@@ -179,6 +179,20 @@ STATIC_INLINE int jl_array_ndimwords(uint32_t ndims)
     return (ndims < 3 ? 0 : ndims-2);
 }
 
+typedef struct {
+    union {
```

Comment:
you can't just randomly change information that codegen has decided to cache here and expect the rest of the codegen system to hold together.